pkg/controller/servicebindingrequest/servicebinder_test.go

```diff
@@ -72,6 +71,39 @@ func TestServiceBinder_Bind(t *testing.T) {
 		wantResult *reconcile.Result
 	}
 
+	assertSchemaPathResolution := func(args args, sb serviceBinder) func(*testing.T) {
+		return func(t *testing.T) {
```

Comment:
Not sure what should be changed here.

Reply:
If you follow the test, you will read that the only test happening here is whether `serviceBinder` and `args` have the same structure, and indeed they should since `sb.sbr` is only a reference to `args.options.sbr`.

@pedjak is right, we should have proper tests for `SetDefaults()` instead.

Reply:
Done. 7531308083ed1f3befdf486f2ae357060b182003

Reply:
This assertion can be removed since `args.options.sbr` and `sb.sbr` are the same reference and `sbr.Spec.ApplicationSelector` is modified in place.

Reply:
Done. 6859f3fe621e7a72142d9815f3219baffe813f9a